cinnabar/plotting.py

```diff
@@ -167,8 +172,15 @@ def _master_plot(
     # stats and title
     statistics_string = ""
     for statistic in statistics:
-        s = stats.bootstrap_statistic(x, y, xerr, yerr, statistic=statistic)
-        string = f"{statistic}:   {s['mle']:.2f} [95%: {s['low']:.2f}, {s['high']:.2f}] " + "\n"
+        s = stats.bootstrap_statistic(x,
+                                      y,
+                                      xerr,
+                                      yerr,
+                                      statistic=statistic,
+                                      include_true_uncertainty=bootstrap_x_uncertainty,
+                                      include_pred_uncertainty=bootstrap_y_uncertainty)
+        statistic_type = 'mean' if bootstrap_x_uncertainty or bootstrap_y_uncertainty else 'mle'
```

Comment:
Instead of auto-selecting the mean or mle depending on whether bootstrap_{x|y}_uncertainty is specified, can we just have a separate kwarg for `statistic_type` that defaults to `mle` and allows the user to set it to `mean` if they prefer? That would make the behavior much more obvious.